pkg/storage/ingest/pusher.go

```diff
@@ -267,11 +287,6 @@ func (c multiTenantPusher) Close() []error {
 	return errs
 }
 
-type shardedPush struct {
-	mimirpb.PreallocTimeseries
-	context.Context
-}
-
```

Comment:
that's unused now, but there was a TODO referring to it. I'm fine removing it now, but FYI in case that comment is then confusing

https://github.com/grafana/mimir/blob/37c2780a1197d1769d07f2bbe8644ce41a18bda5/pkg/storage/ingest/pusher.go#L368